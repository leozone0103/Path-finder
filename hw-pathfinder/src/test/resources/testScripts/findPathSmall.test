# test relatively small amount of data
CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D
AddNode graph E

AddEdge graph A B 2.000
AddEdge graph A C 4.000
AddEdge graph B C 3.000
AddEdge graph B D 1.000
AddEdge graph D C 1.000
AddEdge graph C D 1.000
AddEdge graph D E 2.000
AddEdge graph C E 5.000


FindPath graph A E
FindPath graph A D
FindPath graph B C
FindPath graph C E
FindPath graph C B
