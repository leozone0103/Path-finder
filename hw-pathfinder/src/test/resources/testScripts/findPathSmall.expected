# test relatively small amount of data
created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph
added node E to graph

added edge 2.000 from A to B in graph
added edge 4.000 from A to C in graph
added edge 3.000 from B to C in graph
added edge 1.000 from B to D in graph
added edge 1.000 from D to C in graph
added edge 1.000 from C to D in graph
added edge 2.000 from D to E in graph
added edge 5.000 from C to E in graph


path from A to E:
A to B with weight 2.000
B to D with weight 1.000
D to E with weight 2.000
total cost: 5.000
path from A to D:
A to B with weight 2.000
B to D with weight 1.000
total cost: 3.000
path from B to C:
B to D with weight 1.000
D to C with weight 1.000
total cost: 2.000
path from C to E:
C to D with weight 1.000
D to E with weight 2.000
total cost: 3.000
path from C to B: